README.md

```diff
@@ -2,20 +2,112 @@
 
 ## Dependencies
 
-#### Java JDK 11+
-[Main website](https://jdk.java.net/)
+#### Java
+[JDK 11+](https://jdk.java.net/) (OpenJDK recommended)
 
 #### Other Dependencies
-See the `dependencies` entry in the [Gradle build file](build.gradle). Gradle should be used to acquire these libraries automatically.
+See the `dependencies` entry in the [Gradle build file](build.gradle). Gradle should be used to automatically download these libraries.
 
-## Configuration
-Config file documentation is available [here](src/main/resources/network/brightspots/rcv/config_file_documentation.txt).
+## Overview
 
-## User Guide
-User Guide is available [here](UserGuide.txt).
+The Universal Tabulator is a free, open-source application designed to quickly and accurately tabulate a wide variety of ranked choice voting elections.
+
+The Tabulator requires a "contest configuration" file as input, which specifies:
+- Which tabulation rule variations to use
+- A list of registered candidates 
+- One or more cast vote record files
+- Output formatting options (contest name, where to write output files, etc.)
+
+The Tabulator produces the following as output:
+- A results spreadsheet including round-by-round vote totals for each candidate and the eventual winner(s)
+- A JSON file containing the same information
+- An audit log describing how every ballot was counted in each round over the course of the tabulation
+
+The Tabulator includes the following functionality:
```

Comment:
I'd move this paragraph up so it follows immediately after "The Universal Tabulator is a free..."